### PR TITLE
Views

### PR DESCRIPTION
**Pull request by kennyegun24**
### In this PR I created the view files to the respective url routes and I populated each of the pages while querying data from the database

### CODE REVIEW

**Review by TadesseDev — COMMENTED** [review]
### Hi @kennyegun24 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑🛑
yPkfDFf

You have submitted a project that has linter tests falling. Please make sure all linters are set up and passing before your next review request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+            </div>
+            <div class='bottom_flex'>
+                <%= "Number of posts: #{@user.PostCounter} " %>
+            </div>
+        </div>
+    </div>
+    <div class='border'>
+        <p>Bio</p>
+        <%= @user.Bio %>
+    </div>
```
Comment (muneebulrehman) [inline app/views/users/show.html.erb:18]: In the user-show page, each post should mention the number of comments and likes as per the wireframe. Also it should show what number the post is. Kindly add it accordingly. 
| Your design  | Wireframe |
| ------------- | ------------- |
| image | image |

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,2 +1,43 @@
...
+                            <% end %>
+                            <% end %>
+                        </div>
+                        <% %>
+                        </div>
+                    <% end %>
+                </div>
+
+    </div>
+</div>
```
Comment (muneebulrehman) [inline app/views/posts/index.html.erb]: - [ ] The post-index page, as per the wireframe, it should show the number of comments and likes. In your application, it does show some numbers, but those numbers should correspond to comments and likes which is missing in your case.
- [ ] It shoud show the username of the user who left the comment. 
- [ ] It should show what number each post is.
Kindly fix these issues.

| Your design  | Wireframe |
| ------------- | ------------- |
| image | image |

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @kennyegun24 , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly update the title of the project in your README file. It has a title of your previous project. ⬇️
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (kennyegun24) [reply app/views/users/show.html.erb:18]: About the post sowing the number it is, I think the number only identifies the particular title of the post and not literally the particular number the post is

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (kennyegun24) [reply app/views/posts/index.html.erb]: I have fixed every issue stated

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1,2 +1,44 @@
...
+            </div>
+        </div>
+            </div>
+                    <% @posts.each do |post| %>
+ 
+                       <div class='borBtm'>
+                       <div class='border'>
+                            <div class='flex gap2'>
+                            <p>Post:</p>
+                                <%=link_to post.title, user_post_path(@user.id, post.id)%>
```
Comment (Ismailco) [inline app/views/posts/index.html.erb:17]: - [ ] Great job implementing most of the requested changes from the previous reviewer, how ever kindly make sure every post shows the id of the post, for example, the first post should have something like `Post #1`

- From the wireframes:
Screenshot_20230211_115020

- your view looks like this  👇🏼 
Screenshot_20230211_114618

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @kennyegun24, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — APPROVED** [review]
## Hi 🙋 @kennyegun24  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights </img>


- PR is professional ✔️ 
- Linters checks passed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**